### ISSUE TITLE
Compiling for Windows.

### ISSUE THREAD

**Issue by Alex313031** [myfreeer/chrome-pak-customizer#15]
Hello, I have made some changes to pak, like using -O3, and adding color to --help, etc.
I compiled for linux, but now I want to make an updated windows binary. The readme and wiki only has instructions for linux, except the  LGPL 2.1 caveat.

How would I compile pak for windows?

**Comment by myfreeer** [comment]
For branch [develop](https://github.com/myfreeer/chrome-pak-customizer/tree/develop), building system is based on [cmake](https://github.com/myfreeer/chrome-pak-customizer/blob/develop/CMakeLists.txt), examples could be found:
* Compiling with MSVC: [appveyor.bat](https://github.com/myfreeer/chrome-pak-customizer/blob/bfabc033207ebbd6e0017ce99500d3e379a0a3f6/appveyor.bat#L21-L35)
* Compiling with [msys2](https://github.com/msys2) mingw-w64 GCC and ninja: [test.sh](https://github.com/myfreeer/chrome-pak-customizer/blob/bfabc033207ebbd6e0017ce99500d3e379a0a3f6/test.sh#L2-L8)

For experimental branch [3.x](https://github.com/myfreeer/chrome-pak-customizer/tree/3.x), building system is based on [cargo](https://github.com/myfreeer/chrome-pak-customizer/blob/3.x/Cargo.toml),  compiled with [cargo build](https://github.com/myfreeer/chrome-pak-customizer/blob/5036446fc929c1fab2d79ada728ae376ee6ca6fb/.github/workflows/rust.yml#L24) command.

**Comment by Alex313031** [comment]
@myfreeer Whats the difference between the 2x and 3x branch (besides a language change) did you add, for example, better brotli support?

**Comment by myfreeer** [comment]
> better brotli support?

Yes, 3.x could strip the custom header of brotli compressed resource on unpacking, and rebuild it on packing. See https://github.com/myfreeer/chrome-pak-customizer/issues/11#issuecomment-1036977635 for more tech details.
